domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleResponse.java

```diff
@@ -10,6 +11,7 @@
     public static class FullDefence extends SampleResponse<FullDefence> {
         private FullDefenceResponse.DefenceType defenceType = FullDefenceResponse.DefenceType.DISPUTE;
         private String defence = "defence string";
+        private PaymentDeclaration paymentDeclaration = SamplePaymentDeclaration.builder().build();
```

Comment:
Please add method that allows overriding `paymentDeclaration` value e.g. `withPaymentDeclaration`